src/olympia/constants/search.py

```diff
@@ -59,31 +59,3 @@
 for analyzer, languages in SEARCH_ANALYZER_MAP.items():
     for language in languages:
         SEARCH_LANGUAGE_TO_ANALYZER[language] = analyzer
-
-# Which stemmer to use for each language.
-#
-# Note: We use the keys of this dict for supported stop words, also, which is
-# specified as, e.g., '_english_'.
-STEMMER_MAP = {
```

Comment:
It's a drive-by fix because I was looking at the constants, we aren't using that anywhere. Each analyzer already comes with its own stemmer config, there is no need to override this.